packages/encoding/src/readFundingIndices.ts

```diff
@@ -0,0 +1,16 @@
+import { ByteReader } from './ByteReader'
+import { MIN_INT } from './constants'
+import { decodeAssetId } from './decodeAssetId'
+import { FundingIndex } from './OnChainData'
+
+export function readFundingIndices(reader: ByteReader) {
+  const fundingIndicesLength = reader.readNumber(32)
+  const indices: FundingIndex[] = []
+  for (let i = 0; i < fundingIndicesLength; i++) {
+    reader.skip(17)
```

Comment:
`reader.skip` mutates, so we need to call these functions in a set order, right?

Reply:
Yup